contract-ui/tabs/overview/components/BuildYourApp.tsx

```diff
@@ -0,0 +1,58 @@
+import { ChakraNextImage as Image } from "../../../../components/Image";
```

Comment:
@shift4id - No need to use full relative path here - you can just use "components/Image"